graviti/dataframe/dataframe.py

```diff
@@ -0,0 +1,235 @@
+
+        Returns:
+            The instance of the iLocIndexer.
+
+        """
+        return ILocIndexer(self)
+
+    def shape(self) -> Tuple[int, int]:
+        """Return a tuple representing the dimensionality of the DataFrame.
+
+        Return:
+            Shape of the DataFrame.
+
+        Examples
+            >>> df = DataFrame({"col1": [1, 2], "col2": [3, 4]})
+            >>> df.shape
+            (2, 2)
+
+        """
+
+    def size(self) -> int:
+        """Return an int representing the number of elements in this object.
+
+        Return:
+            Size of the DataFrame.
+
+        Examples
+            >>> df = pd.DataFrame({"col1": [1, 2], "col2": [3, 4]})
+            >>> df.size
+            4
+
+        """
+
+    def head(self, number: int = 5) -> "DataFrame":
+        """Return the first `n` rows.
+
+        Arguments:
+            number : Number of rows to select.
+
+        Return:
+            The first `n` rows.
+
+        Examples
+            >>> df = pd.DataFrame({"animal": ["alligator", "bee", "falcon", "lion",
+            ...                    "monkey", "parrot", "shark", "whale", "zebra"]})
+            >>> df
+                  animal
+            0  alligator
+            1        bee
+            2     falcon
+            3       lion
+            4     monkey
+            5     parrot
+            6      shark
+            7      whale
+            8      zebra
+
+            Viewing the first `n` lines (three in this case)
+
+            >>> df.head(3)
+                  animal
+            0  alligator
+            1        bee
+            2     falcon
+
+            For negative values of `n`
+
+            >>> df.head(-3)
+                  animal
+            0  alligator
+            1        bee
+            2     falcon
+            3       lion
+            4     monkey
+            5     parrot
+
+        """
+
+    def tail(self, number: int = 5) -> "DataFrame":
+        """Return the last `n` rows.
+
+        Arguments:
+            number : Number of rows to select.
+
+        Return:
+            The last `n` rows.
+
+        Examples
+            >>> df = pd.DataFrame({"animal": ["alligator", "bee", "falcon", "lion",
+            ...                    "monkey", "parrot", "shark", "whale", "zebra"]})
+            >>> df
+                  animal
+            0  alligator
+            1        bee
+            2     falcon
+            3       lion
+            4     monkey
+            5     parrot
+            6      shark
+            7      whale
+            8      zebra
+
+            Viewing the last 5 lines
+
+            >>> df.tail()
+               animal
+            4  monkey
+            5  parrot
+            6   shark
+            7   whale
+            8   zebra
+
+            Viewing the last `n` lines (three in this case)
+
+            >>> df.tail(3)
+              animal
+            6  shark
+            7  whale
+            8  zebra
```

Comment:
we still need `loc` method  because the dataframe generated by tail whose index may not start from 0